src/Nncase.Transform/Rules/Neutral/Fusion.cs

```diff
@@ -30,20 +30,27 @@ public partial class SingleInputFusion<T, BeginT, EndT> : FusionMaker
     where EndT : Op
 {
     /// <inheritdoc/>
-    public override Pattern Pattern { get; } = IsWildcardCall<EndT>("st", null!,
-        IsWildcardCall<T>(null!, null!,
-            IsWildcardCall<BeginT>(null!, null!, IsWildcard("input"))));
-
-    // replace input with var
-    private Call? GetReplace(Call st, Expr input, RunPassOptions passOptions)
+    public override Pattern Pattern { get; } = IsWildcardCall<EndT>("end_call", "end_op",
+        IsWildcardCall<T>("mid_call", "mid_op",
+            IsWildcardCall<BeginT>("begin_call", "begin_op", IsWildcard("input"))));
```

Comment:
Some of the names that appear in the pattern are not used, I think it is enough to pass only the names used, the others can be filled by null.